feature/converter/converter.py

```diff
@@ -0,0 +1,32 @@
+class Converter:
+
+    def __init__(self, text):
+        self.text = text
+
+    def split_text(self):
+        new_text = []
+        for element in self.text:
+            element = element.strip('\r')
+            string = element.split(',')
```

Comment:
Such string methods as **strip**, **split** return value so you don't have to make a new assignment every time you use different method, e.g. 

`string = element.strip("\r").split(",")`